test/blackbox-tests/test-cases/cinaps/multiple-cinaps.t

```diff
@@ -0,0 +1,34 @@
+Multiple cinaps stanzas in the same dune file
+
+  $ cat > dune-project <<EOF
+  > (lang dune 2.8)
+  > (using cinaps 1.0)
+  > EOF
+
+  $ cat > dune <<EOF
+  > (cinaps (files *.ml))
+  > (cinaps (files *.mli))
+  > EOF
+
+  $ dune runtest --diff-command diff 2>&1 | sed -E 's/[^ ]+sh/\$sh/'
+  File ".cinaps.6c465b1c/cinaps.ml-gen", line 1, characters 9-28:
```

Comment:
What about adding `touch x.ml` and `touch x.mli` here so that we don't hit this cinaps bug?